internal/reconciler/logpipeline/reconciler.go

```diff
@@ -350,45 +350,15 @@ func getFluentBitPorts() []int32 {
 	}
 }
 
-func getTLSCertValidationResult(ctx context.Context, pipeline *telemetryv1alpha1.LogPipeline, validator TLSCertValidator, client client.Client) cert.TLSCertValidationResult {
+func getTLSCertValidationResult(ctx context.Context, pipeline *telemetryv1alpha1.LogPipeline, validator TLSCertValidator) tlscert.TLSCertValidationResult {
 	if pipeline.Spec.Output.HTTP == nil || (pipeline.Spec.Output.HTTP.TLSConfig.Cert == nil && pipeline.Spec.Output.HTTP.TLSConfig.Key == nil) {
-		return cert.TLSCertValidationResult{
+		return tlscert.TLSCertValidationResult{
```

Comment:
Can we avoid returning a dummy value here?

Reply:
fixed